### PR TITLE
Feat administration

### PR DESCRIPTION
**Pull request by Wroud**

### CODE REVIEW

File: webapp/packages/core/src/root/PermissionsService.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  constructor(
+    private graphQLService: GraphQLService,
+  ) { }
+
+  has(id: string): boolean {
+    return this.permissions.data.has(id);
+  }
+
+  async hasAsync(id: string): Promise<boolean> {
```
Comment (dkrupenya) [inline webapp/packages/core/src/root/PermissionsService.ts]: never required. Permissions will be loaded on application bootstrap. there will be no cases in normal workflow when there are no permissions

File: webapp/packages/core/src/app/TopNavBar/SettingsMenu/SettingsMenuService.ts [hunk 2]
```diff
@@ -23,21 +23,21 @@ export class SettingsMenuService {
...
     this.addLocales();
   }
 
   getMenu() {
-    return this.menu.getMenu(this.settingsMenuToken);
+    return this.menu.getMenu(SettingsMenuService.settingsMenuToken);
   }
 
-  addMenuItem(panelId: string, options: IComputedMenuItemOptions) {
+  addMenuItem(options: IComputedMenuItemOptions, panelId: string = SettingsMenuService.settingsMenuToken) {
```
Comment (dkrupenya) [inline webapp/packages/core/src/app/TopNavBar/SettingsMenu/SettingsMenuService.ts]: revert back.
all other `addMenuItem` has signature "panel, option"

**Review by dkrupenya — CHANGES_REQUESTED** [review]

File: webapp/packages/core/src/root/PermissionsService.ts (hunk 1 above)
Comment (Wroud) [reply webapp/packages/core/src/root/PermissionsService.ts]: fixed

File: webapp/packages/core/src/app/TopNavBar/SettingsMenu/SettingsMenuService.ts (hunk 2 above)
Comment (Wroud) [reply webapp/packages/core/src/app/TopNavBar/SettingsMenu/SettingsMenuService.ts]: fixed

**Review by dkrupenya — APPROVED** [review]

**Pull request merged**